render/Driver3D_OGL.cpp

```diff
@@ -66,7 +66,7 @@ Driver3D_OGL::~Driver3D_OGL()
 		glDeleteTextures( (GLsizei)m_uTextureCnt, (GLuint *)m_Textures );
 	}
 
-	IDriver3D::~IDriver3D();
+	// IDriver3D::~IDriver3D();
```

Comment:
Just remove this, no reason to comment it out.

Reply:
Just wanted to leave a trace in case of runtime errors, will delete that.